src/style.ts

```diff
@@ -0,0 +1,34 @@
+import * as prebase from './prebase';
+
+export enum Enclosure {
+    // noinspection JSUnusedGlobalSymbols
+    RECTANGLE = 'rectangle',
+    SQUARE = 'square',
+    OVAL = 'oval',
+    CIRCLE = 'circle',
+    BRACKET = 'bracket',
+    TRIANGLE = 'triangle',
+    DIAMOND = 'diamond',
+    PENTAGON = 'pentagon',
+    HEXAGON = 'hexagon',
+    HEPTAGON = 'heptagon',
+    OCTAGON = 'octagon',
+    NONAGON = 'nonagon',
+    DECAGON = 'decagon',
+    NONE = 'none',  // special.  sets to undefined
+}
+
+
+// noinspection JSUnusedGlobalSymbols
+export class Style extends prebase.ProtoM21Object {
+    size: undefined;
+    relativeX: number|undefined;
+    relativeY: number|undefined;
+    absoluteX: number|undefined;
+    absoluteY: number|undefined;
+    enclosure: Enclosure|undefined;
+    // fontRepresentation
+    color: string|undefined;
```

Comment:
No issue with this. Just asking for when it comes up in future: should we be adding `|undefined` like this in the event we turn on ``--strict`` in the ts compiler? Right now with strict=false, undefined is always implied. (And we're a long way from turning strict mode on.)

Reply:
I want to do this and eventually turn on `--strict`.  We're a long way from turning it on, but I want to show places where undefined is a totally acceptable value and not just a "we haven't gotten around to defining it yet"